examples/gaussian_cloning.py

```diff
@@ -21,8 +20,8 @@
     BS | (q[1], q[2])
     MeasureX | q[1]
     MeasureP | q[2]
-    Xgate(scale(q[1], sqrt(2))) | q[0]
-    Zgate(scale(q[2], sqrt(2))) | q[0]
+    Xgate(q[1].par * sqrt(2)) | q[0]
+    Zgate(q[2].par * sqrt(2)) | q[0]
```

Comment:
While going through my release checklist, I noticed that this particular example was outdated and wouldn't work with the existing SF version.